SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorTaskBuilder.java

```diff
@@ -55,7 +56,7 @@ public SingularityExecutorTaskBuilder(ObjectMapper jsonObjectMapper, JsonObjectF
   }
 
   public Logger buildTaskLogger(String taskId) {
-    Path javaExecutorLogPath = configuration.getTaskDirectoryPath(taskId).resolve(configuration.getExecutorJavaLog());
+    Path javaExecutorLogPath = MesosUtils.getTaskDirectoryPath(taskId).resolve(configuration.getExecutorJavaLog());
```

Comment:
like i said in `MesosUtils`, we shouldn't be implicitly basing all this off the current directory -- if possible, let's build up an absolute path (i know this isn't your fault but let's take a stab at improving this)